terraform/node_resource_refresh.go

```diff
@@ -176,3 +172,77 @@ func (n *NodeRefreshableManagedResourceInstance) evalTreeManagedResource() EvalN
 		},
 	}
 }
+
+func (n *NodeRefreshableManagedResourceInstance) evalTreeManagedScaleOutResource() EvalNode {
```

Comment:
I'm not married to the function name, so yeah, we can change this no problems. :slightly_smiling_face:  I can even comment the function so that we know at least one example use case.